parm/config/gefs/config.base

```diff
@@ -131,6 +131,7 @@ export APP=@APP@
 export DO_ATM="YES"
 export DO_COUPLED="NO"
 export DO_WAVE="NO"
+export DISABLE_WAVE="@DISABLE_WAVE@"
```

Comment:
I tend to agree w/ @WalterKolczynski-NOAA 
Rather than `DISABLE_WAVE`, one could employ some kind of validation in `setup_expt.py`  based on the input value of `--app` argument.

Reply:
That's a good idea @aerorahul.  I'll add that in.